content/_releases/22/14.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Product Update - v22.14
+description: Details of the product update v22.14 from 11th of April 2022.
+layout: rel
+version: "22.14"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-04-11
+redirect_from:
+  - /releases/22.14.html
+---
+
+## Improvements and Updates
+
+### Step designer: Sample data part
+
+To provide a unified experience during the flow step design, we improved the Sample
+data retrieval section. The look and feel is now more in line with our new
+minimalist approach.
+
+![Step designer: Sample Data part](/assets/img/RN/2214/step-designer-sample-data-initial.png)
+
+The above images show the new Sample data retrieval view. The release {{page.version}}
+add a new functionality, now you can skip the sample generation by clicking on
+the **Skip Sample** button. The platform will add an empty JSON `{}` instead of
+the Sample data.
+
+When you retrieve the Sample the platform UI will show it only in JSON format. In
```

Comment:
In version
{{page.version}}  we removed the Integrator view of Data Sample.

Reply:
OK